sqlagg/columns.py

```diff
@@ -161,3 +186,19 @@ def _build_whens(self):
             then = text(then) if isinstance(then, six.string_types) else then
             whens.append((when, then))
         return whens
+
+
+class ArrayAggLastValueColumn(SqlColumn):
```

Comment:
Consider renaming to just `ArrayAggColumn` since the 'last value' part isn't part of this class.

```suggestion
class ArrayAggColumn(SqlColumn):
```